test/sql/test_generate_series/R/test_generate_time_series

```diff
@@ -0,0 +1,135 @@
+-- name: generate_series_by_seconds_01
+select * from table(generate_series('2023-12-31 23:59:56', '2024-01-01 00:00:06', interval 3 second));
```

Comment:
add some abnormal cases:
1. interval as columns with null value
2. begin/end also can be null value in a column
3. what if invalid datetime? like 0, -00:00, -2025-02-30...
4. what if float interlval?

Reply:
added test case
1. will use default 1
2. will return empty row;
3. will return empty row;
4. currently step only support integer